cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -988,6 +989,33 @@ def ceph_storage_privilege():
     reconfigure_apiserver(endpoint_from_flag('etcd.available'))
 
 
+@when('kubernetes-master.ceph.configured')
+@when_not('kubernetes-master.ceph.pool.created')
+def ceph_storage_pool():
+    '''Once Ceph relation is ready,
```

Comment:
This is only necessary for CNI versions >=1.12.

Reply:
I might have misunderstood, but I don't see where this is done in `ceph_storage`; that's only setting up CSI or the Kube secret, right?

Reply:
It isn't done in ceph_storage, but it isn't needed. The CSI has the expectations of pools. I believe the original ceph implementation just setup communication and allowed creation of pools and persistent volumes via actions. This means there is no need to automagically generate a pool for those clusters.

Reply:
Ahhh, gotcha